.github/CODEOWNERS

```diff
@@ -0,0 +1 @@
+* @camilasan @claucambra @mgallien @nilsding
```

Comment:
ideally we could name a GitHub team for that (as it's done in e.g. nextcloud/server), but the [Desktop team in the organisation](https://github.com/orgs/nextcloud/teams/desktop) contains more than just ourselves